server/src/main/java/org/elasticsearch/discovery/PeerFinder.java

```diff
@@ -287,20 +286,9 @@ private boolean handleWakeUp() {
             }
         });
 
-        transportService.getThreadPool().scheduleUnlessShuttingDown(findPeersInterval, Names.GENERIC, new AbstractRunnable() {
+        transportService.getThreadPool().scheduleUnlessShuttingDown(findPeersInterval, Names.CLUSTER_COORDINATION, new Runnable() {
             @Override
-            public boolean isForceExecution() {
```

Comment:
No need for a force execution here (and really never has been with the generic pool that didn't use a sized queue either), all this does is trip a check if there is an actual rejection on submit to a shutdown pool.